core/src/main/java/com/digitalpebble/storm/crawler/filtering/depth/MaxDepthFilter.java

```diff
@@ -0,0 +1,52 @@
+package com.digitalpebble.storm.crawler.filtering.depth;
```

Comment:
indeed. Eclipse is pretty good at suggesting / autocompleting so it's not like we have to type it very often